lib/detect-testing-library-utils.ts

```diff
@@ -580,14 +581,25 @@ export function detectTestingLibraryUtils<
       return isRenderUtil(initIdentifierNode);
     };
 
-    const isDebugUtil: IsDebugUtilFn = (node) => {
-      return isTestingLibraryUtil(
-        node,
-        (identifierNodeName, originalNodeName) => {
-          return [identifierNodeName, originalNodeName]
-            .filter(Boolean)
-            .includes('debug');
-        }
+    const isDebugUtil: IsDebugUtilFn = (identifierNode) => {
+      const isBuiltInConsole =
+        isMemberExpression(identifierNode.parent) &&
+        identifierNode.parent.parent &&
+        isCallExpression(identifierNode.parent.parent) &&
```

Comment:
We aren't interested in checking if the received identifier comes from a Call Expression, that's up to whichever code is calling `isDebugUtil`. This should only check if the identifier itself represents a debug, discarding those belonging to a Member Expression with "console object. Can you remove this check please?

Reply:
I actually needed to know thee parent is a CallExpression because that's the only way to differentiate `console.debug()` from plain `debug()`. For the first case, I know I can ignore it. For the 2nd one, I need further info, which are the following lines of code